searx/webapp.py

```diff
@@ -377,7 +377,7 @@ def _get_enable_categories(all_categories: Iterable[str]):
         for category in engines[engine_name].categories
         if (engine_name, category) not in disabled_engines
     )
-    return [x for x in all_categories if x in enabled_categories]
+    return [x for x in ui_categories if x in enabled_categories]
```

Comment:
Is we change the name (I'm not sure why), why not `categories_as_tabs` which seems more relevant?

Reply:
This is just a list of names generated by the caller `settings['categories_as_tabs'].keys()` .. I dont wanted to mix this list with the value of `settings['categories_as_tabs']`.

May it is better the function does not have this argument and uses `settings['categories_as_tabs'].keys()` directly.

Reply:
I've commit for this:
![image](https://user-images.githubusercontent.com/1594191/182177983-9ec1f621-b4b6-4d09-9044-221cb1e0c898.png)

But this is misleading: actually `!web` includes engines in the general, and videos categories, so if we document the bang we should list the engines too (in the tooltip).